<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1214</width>
    <height>657</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Main Application Window</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QScrollArea" name="wrapperScrollArea">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget#wrapperScrollAreaWidgetContents{
/*background-color: rgb(0,0,0);*/
}


QGroupBox#Route{
 background-color: rgb(233, 247, 215);
 border: 3px solid black; 
 border-radius: 5px; 
}

QGroupBox#originalRoute{
 background-color: rgb(233, 247, 215);
 border: 3px solid black; 
 border-radius: 5px; 
}

QGroupBox#subRoute{
 background-color: rgb(221, 242, 191);
 border: 1px solid black; 
}

#wrapperScrollArea{
 background-color:rgb(255,255,255);
}</string>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="wrapperScrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1194</width>
         <height>535</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>1250</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="nextPushButton">
         <property name="text">
          <string>Next</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="generateRoutePushButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Generate Route</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1214</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="importXMLFileAction"/>
    <addaction name="exportXMLFileAction"/>
    <addaction name="separator"/>
    <addaction name="exitAction"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="importXMLFileAction">
   <property name="icon">
    <iconset>
     <normaloff>../../../../../../Users/RMPLplus/Desktop/RACHEL APP/import3.jpg</normaloff>../../../../../../Users/RMPLplus/Desktop/RACHEL APP/import3.jpg</iconset>
   </property>
   <property name="text">
    <string>Import XML</string>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
  </action>
  <action name="exportXMLFileAction">
   <property name="icon">
    <iconset>
     <normaloff>../../../../../../Users/RMPLplus/Desktop/RACHEL APP/export2.jpg</normaloff>../../../../../../Users/RMPLplus/Desktop/RACHEL APP/export2.jpg</iconset>
   </property>
   <property name="text">
    <string>Export XML</string>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
  </action>
  <action name="exitAction">
   <property name="icon">
    <iconset>
     <normaloff>../../../../../../Users/RMPLplus/Desktop/RACHEL APP/close-icon.png</normaloff>../../../../../../Users/RMPLplus/Desktop/RACHEL APP/close-icon.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionTest">
   <property name="text">
    <string>Test</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>wrapperScrollArea</tabstop>
  <tabstop>generateRoutePushButton</tabstop>
  <tabstop>nextPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
